dev/docker/building/README.md

```diff
@@ -0,0 +1,30 @@
+# Gravitino Dev Env Image
+This directory contains dockerfiles to build the docker image for the development environment.
+
+A Gravitino development environment contains all necessary development tools installed as well as prebuilt Gravitino third
+party dependencies.
+
+## 1 Build dev env image
+
+### 1.1 Build dev env image
+```
+DOCKER_BUILDKIT=1 docker build --rm=true -f dev-env.Dockerfile -t ghcr.io/OWNER/gravitino/dev-env:<tag> .
+```
+E.g.:
+```shell
+DOCKER_BUILDKIT=1 docker build --rm=true -f dev-env.Dockerfile -t ghcr.io/gravitino/dev-env:main .
+```
+
+## 2 Publish image to ghcr
+```
+docker push ghcr.io/OWNER/starrocks/dev-env:<tag>
```

Comment:
starrocks?

Reply:
Sorry, I implement it use Starrocks for reference, and also it is too simple and incomplete